src/benchmarklib/abstract_table_generator.cpp

```diff
@@ -407,4 +438,52 @@ std::unordered_map<std::string, BenchmarkTableInfo> AbstractTableGenerator::_loa
   return table_info_by_name;
 }
 
+std::unordered_map<std::string, BenchmarkTableInfo> AbstractTableGenerator::_load_binary_tables_from_json() {
+  std::unordered_map<std::string, BenchmarkTableInfo> table_info_by_name;
+  auto& storage_manager = Hyrise::get().storage_manager;
+
+  std::cout << "-  Loading tables file mapping from storage json. " << std::endl;
+  Timer timer_json_parsing;
+  const auto tables_files_mapping = storage_manager.get_tables_files_mapping();
+  std::cout << " (" << timer_json_parsing.lap_formatted() << ")" << std::endl;
+
+  for (const auto& mapping : tables_files_mapping) {
+    const auto table_name = mapping.first;
+
+    std::cout << "-  Loading table '" << table_name << "' from storage json. " << std::endl;
+    Timer timer;
+
+    const auto persistence_data = mapping.second;
+    auto file_name = persistence_data.file_name;
+    const auto file_count = persistence_data.file_index;
+    auto total_chunks = std::vector<std::shared_ptr<Chunk>>{};
+
+    // Create column definitions
+    const auto column_definitions = storage_manager.get_table_column_definitions_from_json(table_name);
+
+    // Load chunks from files
+    for (auto index = size_t{0}; index < file_count + 1; ++index) {
```

Comment:
Since the `+ 1` is not immediately clear (imo), I'd add a comment why you're using it here.